### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by frizzle101101**

### CODE REVIEW

File: src/overlays/actors/ovl_Boss_Dodongo/z_boss_dodongo.cpp [hunk 1]
```diff
@@ -80,6 +80,10 @@ static Color_RGBA8 dustPrimColor_95 = {255, 255, 100, 255};
 
 static Color_RGBA8 dustEnvColor_95 = {255, 100, 0, 255};
 
+static u8 originalLavaFloorTex[8192] = {
+#include "assets/EUR_MQD/overlays/ovl_Boss_Dodongo/lava_floor_lava.rgba16.inc.c"
+};
+
```
Comment (xoascf) [inline src/overlays/actors/ovl_Boss_Dodongo/z_boss_dodongo.cpp:86]: This will not compile if the user has not extracted the EUR_MQD ROM.

**Comment by xoascf** [comment]
Saria no longer appears (regression for previously fixed bugs: #24, #118).
This only affects when the frame rate value is greater than 20.
NoSariaSad

File: src/overlays/actors/ovl_Boss_Dodongo/z_boss_dodongo.cpp (hunk 1 above)
Comment (frizzle101101) [reply src/overlays/actors/ovl_Boss_Dodongo/z_boss_dodongo.cpp:86]: yup good catch was not intended for commit

**Review by RyzenDew — CHANGES_REQUESTED** [review]
in places beside the tunnel it can still switch back to 20fps

**Comment by RyzenDew** [comment]
image
can't compile

**Comment by frizzle101101** [comment]
> in places beside the tunnel it can still switch back to 20fps

how?

**Comment by xoascf** [comment]
Particles are stuck now.
water

**Comment by xoascf** [comment]
Grass animation is wrong:

https://user-images.githubusercontent.com/28744392/187352096-93470fb2-d189-4a50-981a-d931df511320.mp4

**Comment by xoascf** [comment]
> > in places beside the tunnel it can still switch back to 20fps
> 
> how? can you eloborate, because it cant just switch back to 20fps anymore, the tunnel enter action is the only location where that is so it would always be related to the tunnel or if something else calls it that i dont know about

I think this is no longer the case, it seems to have been fixed in 
8006a36, it used to happen when climbing, sometimes when jumping and opening chests.

**Comment by RyzenDew** [comment]
> > > in places beside the tunnel it can still switch back to 20fps
> > 
> > 
> > how? can you eloborate, because it cant just switch back to 20fps anymore, the tunnel enter action is the only location where that is so it would always be related to the tunnel or if something else calls it that i dont know about
> 
> I think this is no longer the case, it seems to have been fixed in [8006a36](https://github.com/blawar/ooot/commit/8006a360fdcd931890c0e7fb13b8354c4c034ce2), it used to happen when climbing, sometimes when jumping and opening chests.

still happens sometimes but not all the time i did more checking today

**Comment by frizzle101101** [comment]
> > > > in places beside the tunnel it can still switch back to 20fps
> > > 
> > > 
> > > how? can you eloborate, because it cant just switch back to 20fps anymore, the tunnel enter action is the only location where that is so it would always be related to the tunnel or if something else calls it that i dont know about
> > 
> > 
> > I think this is no longer the case, it seems to have been fixed in [8006a36](https://github.com/blawar/ooot/commit/8006a360fdcd931890c0e7fb13b8354c4c034ce2), it used to happen when climbing, sometimes when jumping and opening chests.
> 
> still happens sometimes but not all the time i did more checking today

can you try to find out when or is it random?

**Comment by frizzle101101** [comment]
with the effects still working on it trying to figure out the best spots for things

**Comment by RyzenDew** [comment]
> > > > > in places beside the tunnel it can still switch back to 20fps
> > > > 
> > > > 
> > > > how? can you eloborate, because it cant just switch back to 20fps anymore, the tunnel enter action is the only location where that is so it would always be related to the tunnel or if something else calls it that i dont know about
> > > 
> > > 
> > > I think this is no longer the case, it seems to have been fixed in [8006a36](https://github.com/blawar/ooot/commit/8006a360fdcd931890c0e7fb13b8354c4c034ce2), it used to happen when climbing, sometimes when jumping and opening chests.
> > 
> > 
> > still happens sometimes but not all the time i did more checking today
> 
> can you try to find out when or is it random?

sadly it's just random

**Comment by Lehpumeh** [comment]
Not really a case of reverting from 60 to 20 randomly, but going through the crawlspace in Hyrule Castle into Zelda's Garden's crawlspace loading zone won't revert back to 60. If you press the Z-Target button in the garden it will go back to 60. 

If you crawl back out of the garden it will stay at 20 till you climb something(I had to climb the ledge or the block a couple times) or manually switching back to 60.

Tested at the Dins Fire Great Fairy entrance which is also a crawlspace and it's the same thing so I guess its just going from a crawlspace into a loading zone. I think only those two instances are the only times you go from a crawl space to a loading zone.

**Comment by RyzenDew** [comment]
The whole village is 60fps it doesn't need to go to 20fps also new bugs breaking any object causes weird effects like this 
image
also happens with the rocks breakings and grass

**Review by RyzenDew — APPROVED** [review]

**Comment by Lehpumeh** [comment]
This commit broke a block puzzle in Forest Temple for the third poe sister. https://github.com/frizzle101101/ooot/commit/b9056ca5f1a83feae1055d9fbd668ba9e4e35a88

The blocks act like they are colliding with something on some pushes and when you are able to get them to connect right the Poe won't spawn.

[2022-11-19 19-28-18-1.webm](https://user-images.githubusercontent.com/99471756/202883242-3905ebb7-28ad-45f6-8cc1-4a52fe4ae439.webm)


Discord User Gore#6743's save 
[oot.zip](https://github.com/blawar/ooot/files/10048469/oot.zip)

**Comment by RyzenDew** [comment]
image
can't build anymore

**Review by RyzenDew — CHANGES_REQUESTED** [review]
This does not fix the build issues requesting a proper fix please thanks

**Comment by RyzenDew** [comment]
These are great fixes and i am merging now due to silence i hope you are ok.

**Comment by frizzle101101** [comment]
yes sorry gotten busy with other things, i dont think i had more to add, the raphnet stuff after [6187405] could be cleaned up with the hid library and where that is located and how it is used but i wasn't sure what the right way to do that was, for [f3d9427], [8c39534] they need to remain and should still be good

personally ive done enough for native 60fps+, and with the solutions for frame interpolation deployed on the other project its just not feeling like native 60 support is something worth doing because it takes away from what the large community that continuously keep putting life into oot out there on the internet through speed running, and they all use the other project. Not only that but native ruins the massive part glitches play in people's interest to continue playing the game because none of them will work and the timing makes glitches that much more impossible than what it is. unless someone can commit to it for a good chunk of time with a community behind them i really dont think its worth the effort as the other port soh is much farther and has a tone of features i wanted to get to, their community is much larger and their passion is that much more, i just went over there and did some checking out then moved on from oot to learn some unity stuff, but as far as soh don't hate as they know what's up and it can actually play on the switch which really was my end goal, id suggest going and checking out the community, there's lots of them but they are very welcoming to our group

**Pull request merged**